packages/kbn-check-saved-objects-cli/current_mappings.json

```diff
@@ -3794,6 +3794,14 @@
       }
     }
   },
+  "significant-events-prompts": {
+    "dynamic": false,
+    "properties": {
+      "type": {
+        "type": "keyword"
+      }
+    }
+  },
```

Comment:
I think you should be able to revert changes in `packages/kbn-check-saved-objects-cli` since this is a new type. It seems you had mappings defined for this type and CI updated our snapshot and then they were removed?